common/djangoapps/third_party_auth/tests/specs/base.py

```diff
@@ -432,7 +458,7 @@ def test_full_pipeline_succeeds_for_linking_account(self):
 
         # First we expect that we're in the unlinked state, and that there
         # really is no association in the backend.
-        self.assert_dashboard_response_looks_correct(student_views.dashboard(request), request.user, linked=False)
```

Comment:
Please delete `assert_dashboard_response_looks_correct` method since it is no longer used.